template.yml

```diff
@@ -127,6 +133,36 @@ Resources:
                 responses: {}
                 httpMethod: POST
                 type: aws_proxy
+            put:
+              summary: Updates an existing User
+              parameters:
+                - in: path
+                  name: username
+                  schema:
+                    type: string
+                  required: true
+              requestBody:
+                required: true
+                content:
+                  application/json:
+                    schema:
+                      $ref: '#/components/schemas/User'
+              security:
+                - CognitoUserPool: []
+              responses:
+                '202':
+                  description: Successful request for updating the user. User will be eventually updated
+                  headers:
+                    Location:
+                      schema:
+                        type: string
+                      description: Relative path to the user URI
```

Comment:
If you want absolute path you probably need to provide the service url as an env variable to the function.

Reply:
Yeah. I do not have that for now and relative path is also a valid option. I prefer absolute paths but I will fix that when I get a static address.